model/src/main/kotlin/OrtResult.kt

```diff
@@ -96,35 +96,22 @@ data class OrtResult(
         )
     }
 
-    /** An object that can be used to navigate the dependency information contained in this result. */
```

Comment:
Commit message nit: Is sorting really refactoring? Maybe rather "chore" / "style"?

Reply:
I think "chore" fits best.